var/spack/repos/builtin/packages/py-meldmd/package.py

```diff
@@ -1,28 +1,88 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
+
 from spack.package import *
 
 
-class PyMeldmd(PythonPackage, CudaPackage):
+class PyMeldmd(CMakePackage, PythonPackage, CudaPackage):
```

Comment:
```suggestion
class PyMeldmd(CMakePackage, PythonExtension, CudaPackage):
```
If this doesn't use the PythonPackage builder and only uses the extends/import tests, then PythonExtension is what you want.